README.md

```diff
@@ -29,16 +31,20 @@ term = Msgpax.unpack!(iodata)
 # => [300, "Spartans"]
 ```
 
-#### Stream-oriented deserialization
+#### Partial deserialization
```

Comment:
If we're going with "Partial" (seems better than old) we should do the same change in `unpack_slice` doc. :bowtie: